### ISSUE TITLE
Add logo to show in integratio HA

### ISSUE THREAD

**Issue by pepsonEL** [thomasgermain/vaillant-component#34]
Hi
Good option will be add LOGO to show it in integration when we add it to intergration
https://toppng.com/uploads/preview/vaillant-vector-logo-free-download-11574016005et3js5waey.png

**Comment by thomasgermain** [comment]
Hi,

yes, I have to do a PR here: https://github.com/home-assistant/brands.  

I still have to look for the requirements of the image (size, format, etc)

**Comment by pepsonEL** [comment]
1. logo.png 512x172
2. logo@2x.png  1022x344
3. icon.png 256x256
4. icon@2x.png  512x512
4.

**Comment by pepsonEL** [comment]
icon i think it is good
https://is2-ssl.mzstatic.com/image/thumb/Purple124/v4/68/56/5f/68565fe0-61cf-ceff-025f-9d8df7a6abc1/AppIcon-0-0-1x_U007emarketing-0-0-0-8-0-0-85-220.png/1200x630wa.png

**Comment by thomasgermain** [comment]
ah yeah, I remember know why I don't do it yet, I need to know the name of the integration. For now this is `vaillant`, but I don't think this is a good name, I think it should be `multimatic`.

I already searched for an image and I found that:
![multimatic](https://user-images.githubusercontent.com/12560542/103001105-a2f86f00-452c-11eb-9b87-6bcb89965c4f.png)

**Comment by pepsonEL** [comment]
Yes i also think that name should be multimatic. Also image for icon is good but for logo maybe only image with name: vaillant should be better.

**Comment by pepsonEL** [comment]
And you know any date when add it to official integration in HA ? Any plan ?

**Comment by thomasgermain** [comment]
Yeah I need to find some time 😞 

I'm still testing a new version using `DataUpdateCoordinator`. It seems to be ok.

Maybe I can release a new version by the end of the year (or at least end of the holiday), switching the name to `multimatic` and adding the images.

Once the integration is renamed I think I can do the PR to HA, but I guess I will have some rework to do :)

**Comment by pepsonEL** [comment]
You think about end of this year 2020 ? If you need tester i can test it. I have a boiler VSC206 with multimatic 700 and VR920.

**Comment by thomasgermain** [comment]
yes, but I'm speaking about the release, not the merge inside HA which depends on the review

**Comment by pepsonEL** [comment]
ok yes i know. Ok i wait for new version which i can test.... Thanks for good job.

**Comment by thomasgermain** [comment]
you can check release https://github.com/thomasgermain/vaillant-component/releases/tag/1.5.0b1 (please note, there are plenty of breaking changes).

I also had a look for logo/icon. But I need either to do  an official PR for the integration or another PR in wheels-custom-integrations in order to do the PR for the logo.., so I will do the PR for logo when integration will be included inside HA

**Comment by pepsonEL** [comment]
ok good job. I start test new version. Remove old and add new.

**Comment by pepsonEL** [comment]
In next release change also name arch zip to multimatic-component.

**Comment by thomasgermain** [comment]
I don't think I can, since it takes the name of the repo. And github doesn't recommend to change the name of the repo

**Comment by pepsonEL** [comment]
Aha ok it is small issue :)

**Comment by pepsonEL** [comment]
remove this:
BREAKING CHANGE: entity_id doesn't contain the domain anymore (e.g: climate.vaillant_bathroom become climate.bathroom)

was no good because when we add entity to lovelace when we have in name entity "multimatic" it was bettrer to search entity.... Now is no friendly.

**Comment by thomasgermain** [comment]
yes, I understand, but I read somewhere (I can't find it back) that this is not recommended to add the integration domain name inside the entity id.

**Comment by pepsonEL** [comment]
Ok i understand. I test version 1.5 and now works ok.

**Comment by thomasgermain** [comment]
I'm also busy testing it, I found some small issue, but it seems to be fine. I'm waiting for someone to test the fan (ventilation)

**Comment by pepsonEL** [comment]
Ok

**Comment by apfelmaennchen** [comment]
> I'm also busy testing it, I found some small issue, but it seems to be fine. I'm waiting for someone to test the fan (ventilation)

I am happy to test the fan/ventilation, but the respective entity doesn't show up in homeassistant ...

**Comment by johanvh83** [comment]
Hi Thomas,
Just installed the integration. Indeed, logo is missing is the integration but all the rest works as a glance. Absolutely amazing job! Great. Many thanks! Could find how to send a private message, so do it this way.
![Snap 2020-12-30 at 13 12 33](https://user-images.githubusercontent.com/76775686/103350574-bfeaf000-4aa0-11eb-82ca-ff8488989e2f.jpg)

**Comment by thomasgermain** [comment]
Hi,

i'm glad it can help you 😃, by the way, do you have any humidity sensor (vr 51) ? Do they show up in HA ? (I don't have VR51, so I'm not able to test that)

**Comment by johanvh83** [comment]
Hi Thomas,

I have 3 VR51 units and indeed, they pop up.

My knowledge is very basic but if I can help you (for example Teamviewer
Remote session), you are welcome!

It would be an honour!

Kind regards,
Johan

P.S. I'm also from Belgium. Do you speak Flemish or Frensh?


Op wo 30 dec. 2020 13:21 schreef Thomas Germain <notifications@github.com>:

**Comment by johanvh83** [comment]
Hi Thomas,

Enjoy your last day of the year.

I have already a question. I tried to make some automations but so far I
couldn't succeed.

First I called up the state. This is for example a smart head in a room
'dressing':

[image: Schermafbeelding 2020-12-31 om 10.33.37.png]

I'd like to change something; temperature or vaillant_mode. How is this
possible?

Into the developer tools -> services, the services are very limited:
[image: Schermafbeelding 2020-12-31 om 10.33.07.png]

So into the configuration.yaml I created a new service:

# Vaillant services
vaillant:
  - platform: vaillant
    service_name: vaillant.set_temperature

Restarted HA, but with the above configuration the service
'vaillant.set_temperature' doesn't pop up in the develop tools -> service.

Can you give me a jump in the right direction?

Many thanks.

Wish you already a happy New Year!

Greetings,
Johan.

P.S. The humidity from the VR51 cannot be seen ;-), it acts as a VR50 now




Op wo 30 dec. 2020 om 13:25 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by thomasgermain** [comment]
Hi,

I'm french speaking. 

You don't have to create any new services, you can use `climate.set_temperature` (you can also use set_hvac_mode, set_preset_mode, turn_off and turn_on; there are common HA services). You also have some specific services: https://github.com/thomasgermain/vaillant-component#provided-services

You should also be able to control mode and temperature using climate card:
- If you change the temperature directly on the card, it's creating a quick veto
- You can click on the calendar icon (= AUTO mode) or the off button (= OFF mode)
- You can click on the 3 dots (upper right of the card) and you can set plenty of presets from there.


May I ask you to create another issue  -> https://github.com/thomasgermain/vaillant-component/issues/new if you have new questions ?

Thanks and happy new year already 🎉 

Thomas

**Comment by johanvh83** [comment]
Hi Thomas,

Sorry to disturb you again.

I found the 'build in' services.

I try to change the mode from 'off' to 'manual' using the automations:

[image: Schermafbeelding 2021-01-02 om 18.10.54.png]
Can you help me to tell me what I have to enter in the 'Service data' lines?
I tried many possibilities but none of them are working.

Thanks in advance for your help!

Johan


Op do 31 dec. 2020 om 10:55 schreef Thomas Germain <notifications@github.com

**Comment by johanvh83** [comment]
Hi,

Please ignore my previous email.

Found the solution:
[image: Schermafbeelding 2021-01-02 om 18.27.07.png]
Enjoying all possibilities!
It's really awesome!

Before my heating was re-programmed and not flexible now almost everything
is possible!
100 times Thanks you!

Greetings,
Johan

Op za 2 jan. 2021 om 18:13 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by johanvh83** [comment]
Hi Thomas,

Into the automations I have below configuration created by the UI:

- service: climate.set_hvac_mode

    data:

      hvac_mode: auto

    entity_id: climate.vaillant_dressing

  - service: climate.set_preset_mode

    data:

      preset_mode: manual

    entity_id: climate.vaillant_hobbykamer

  - service: climate.set_temperature

    data:

      temperature: 14.5

    entity_id: climate.vaillant_hobbykamer


When I fire this automation the 'hobbykamer' room which was set to 'OFF':


hvac_modes:
  - auto
  - 'off'
min_temp: 5
max_temp: 30
preset_modes:
  - comfort
  - manual
  - system_off
current_temperature: 12.1
temperature: 5
hvac_action: idle
preset_mode: none
vaillant_mode: 'OFF'
friendly_name: Hobbykamer
supported_features: 17


Changes to:


hvac_modes:
  - auto
  - 'off'
min_temp: 5
max_temp: 30
preset_modes:
  - comfort
  - manual
  - system_off
current_temperature: 12.2
temperature: 11.5
hvac_action: idle
preset_mode: manual
vaillant_mode: MANUAL
friendly_name: Hobbykamer
supported_features: 17


So this is correct what I want, the room change to MANUAL with 11.5 temp
(as test ;-)

but unfortunately at the same time room 'dressing':
climate.vaillant_dressing also changed


from state 'OFF':


hvac_modes:
  - auto
  - 'off'
min_temp: 5
max_temp: 30
preset_modes:
  - comfort
  - manual
  - system_off
current_temperature: 14.8
temperature: 5
hvac_action: idle
preset_mode: none
vaillant_mode: 'OFF'
friendly_name: Dressing
supported_features: 17


to:


hvac_modes:
  - auto
  - 'off'
min_temp: 5
max_temp: 30
preset_modes:
  - comfort
  - manual
  - system_off
current_temperature: 14.9
temperature: 14
hvac_action: idle
preset_mode: comfort
vaillant_mode: AUTO
friendly_name: Dressing
supported_features: 17


to 'AUTO' with temp 14. There is only 1 automations setup so it's not
possible something else triggers the Dressing room.

Any idea why the 'dressing' room changes while I only change 'Hobbykamer'?


Thanks for your help!

Johan




Op za 2 jan. 2021 om 18:29 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by johanvh83** [comment]
Sorry,

temperature: 14 should be temperature: 11.5, this is a mistake from my side
but this is correct.

Op zo 3 jan. 2021 om 21:51 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by johanvh83** [comment]
APOLOGIES.

Please ignore last 2 emails, found the problem.

APOLOGIES

Op zo 3 jan. 2021 om 22:00 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by johanvh83** [comment]
Hi Thomas,

Is there an easy way to check/control the communication between HA and
Vaillant? I'm running HA supervised on a Synology NAS in Docker.

I'm confused with the operations and have sometimes strange unexpected
results.

Can you explain the relation between the modes in Multimatic App 'Off -
Manual - Auto' and the HA status:
hvac_mode
hvac_preset
hvac_action

Spend many hours by changing things and check the state later on but I get
very strange things. Seems there is also a delay and when doing things too
fast, the Vaillant becomes crazy.
I never switched on 'away mode' but my Vaillant system became 'system
off'...

Kind regards,
Johan






Op zo 3 jan. 2021 om 22:02 schreef Johan Van Hoorde <johanvh83@gmail.com>:

**Comment by johanvh83** [comment]
Hi Thomas,

One of the 'strange results' in detail:

A room has status: ‘Operation: Auto’ / ‘Preset: ‘Comfort’.
I used a UI Card to change only Preset from 'Comfort' to 'System_off'
(automatically the Operation mode for this room changed from 'Auto' to
'Off').
The change of Preset from Comfort to System off resulted that all UI Cards
became greyscaled and card with ‘water_heater.vaillant_control_dhw’ which
normally has the state ‘AUTO’ became ‘QM_System_off’ (see picture below),
also 'Away mode' was on. Checked Multimatic App and ‘special mode’ -
‘System OFF’ was active. I tried this for 2 rooms but same operation and
result.

[image: Schermafbeelding 2021-01-14 om 12.48.39.png]

I don't know if this is a normal operation. Maybe a bug in the Vaillaint
system of your integration.
Once again: I don't understand all settings so that makes it difficult for
me.

On your page I found:
QM_SYSTEM_OFF -> HVAC_MODE_OFF & PRESET_SYSTEM_OFF (custom)

So it makes sense that QM_SYSTEM_OFF was fired because HVAC_mode and
Preset_System were both OFF, but only for 1 room, not for the entire
heating system. I would suspect that this QM_SYSTEM_OFF is only active when
all rooms and zones have HVAC_mode: OFF and Preset_System: OFF



The Multimatic App has only 3 modes: OFF - AUTO - MANUAL
In HA there are more options:

hvac_mode -> ?
hvac_preset -> This is to set the Multimatic app OFF (= Off) - AUTO (=
Comfort) - Manual (= Manual)
hvac_action -> ?



Op do 14 jan. 2021 om 11:58 schreef Johan Van Hoorde <johanvh83@gmail.com>:
